docs/Deploying.md

```diff
@@ -21,108 +9,55 @@ Use it as a template to create your own:
     cd src/main/resources
     cp portal.properties.EXAMPLE $HOME/cbioportal/portal.properties
 
-For more information about the `portal.properties` file,
-see the [reference](portal.properties-Reference.md) page.
-
-## Set environment variables for Tomcat
-
-The `PORTAL_HOME` environment variable needs to be available to
-the `cbioportal.war` file which runs within the Tomcat server,
-so that it can find the configuration file. To make it available to Tomcat,
-edit your Tomcat startup file (typically `$CATALINA_HOME/bin/setenv.sh`)
-and add a line like the following, pointing it to the folder containing
-`portal.properties`:
+For more information about the `portal.properties` file, see the
+[reference](portal.properties-Reference.md) page.
 
-    export PORTAL_HOME=/Users/johndoe/cbioportal
+Several scripts of cBioPortal use this `portal.properties` file to get info
+like db connection parameters. You can indicate the folder where this file is
+with an environment variable:
 
-In additon, add a line to make Tomcat pass `-Dauthenticate=false`
-as a JVM argument, or replace the word ‘false’ by the method to use:
-
-    CATALINA_OPTS='-Dauthenticate=false'
-
-## Add the MySQL JDBC Driver to Apache Tomcat
-
-A proper JDBC driver will also need to be accessible by Apache Tomcat.  If using MySQL, the [Connector/J](http://dev.mysql.com/downloads/connector/j/) driver jar file should be placed in `$CATALINA_HOME/lib`.
-
-More information on configuring Apache Tomcat connection pooling can be found [here](http://tomcat.apache.org/tomcat-7.0-doc/jndi-datasource-examples-howto.html).
-
-***We have reports that the Tomcat package that comes with (at least) Ubuntu 14.04 cannot handle the connection pool from resources.  If you are encountering this is, we suggest you [download the Tomcat archive from Apache and install from there](https://www.digitalocean.com/community/tutorials/how-to-install-apache-tomcat-8-on-ubuntu-14-04).***
+```
+export PORTAL_HOME=$HOME/cbioportal
+```
 
-## Set up the Database Connection Pool
+if your properties file is at `PORTAL_HOME/portal.properties`
 
-Apache Tomcat provides the database database connection pool to the cBioPortal.  To setup a database connection pool managed by Tomcat, add the following line to `$CATALINA_HOME/conf/context.xml`, making sure that the properties match your system:
+## Run the app
 
-``` xml
-<Context>
```

Comment:
all these settings are already set when using dbcp

Reply:
we do not need to configure maven to download webapp runner?

Reply:
we do not need to configure mvn to download webapp runner?

Reply:
It is downloaded automatically (it's an existing dependency in the pom)

Reply:
I see. great!